docs/spfx/sharepoint-2019-and-subscription-edition-support.md

```diff
@@ -0,0 +1,163 @@
+
+### Prepare the environment for SharePoint Framework development
+
+1. Install [Node.js v8.17.0](https://nodejs.org/download/release/v8.17.0/).<br>
+   SPFx v1.4.1 is also supported on Node.js v12 and v14 (v12.18.1 and v14.17.1 to be specific), although there is an incompatible issue [gulp 3 wasn't compatible with Node 12+](https://github.com/gulpjs/gulp/issues/2324). The workaround to resolve this issue is to specify the version of graceful-fs component as 4+. You can create npm-shrinkwrap.json in the root folder of the project and input the following content, and then run “npm install”. Also, you can use package-lock.json to resolve this issue too.
+
+```powershell
+    { 
+
+      “dependencies”:{ 
+         “graceful-fs”: { 
+                “version”: “4.2.2” 
+          } 
+       } 
+    }
+```   
+
+2. Install global dependencies.<br> 
+   `npm install gulp-cli@2.3.0 --global`<br>
+   `npm install yo@2.0.6 --global`<br>
+   `npm install @microsoft/generator-sharepoint@1.10.0 --global`<br> 
+
+For more information, see [SharePoint Framework development tools and libraries compatibility](tools-and-libraries.md). 
+
+## Build and deploy your first web part with SharePoint Framework 
+
+To create a new web part with SharePoint Framework , see [Build your first SharePoint client-side web part](web-parts/get-started/build-a-hello-world-web-part.md).
+
+To deploy your web part to SharePoint on-premises, unlike deploying to SharePoint Online, some dependent service applications and specific configurations on the SharePoint Server are required, seeing below tutorials. You can contact SharePoint Server administrator if you do not have appropriate permission to check or configure. 
+
+### Create service applications
+
+Ensure the following service applications are enabled on the SharePoint Server: 
+
+- App Management Service
+- Microsoft SharePoint Foundation Subscription Settings Service
+- Managed Metadata Web Service
+
+In Central Admin site, you can create App Management Service application and Managed Metadata Web Service application by selecting **Application Management --> Manage service applications**.
+
+:::image type="content" source="../images/manage-service-application.png" alt-text="This is manage service application image.":::
+
+To create SharePoint Foundation Subscription Settings Service, use the following PowerShell command:
+
+```powershell
+$sa = New-SPSubscriptionSettingsServiceApplication -ApplicationPool $applicationPoolName -Name $serviceApplicationName -DatabaseName $dataBaseName 
+
+New-SPSubscriptionSettingsServiceApplicationProxy -ServiceApplication $sa
+```
+
+### Prepare .sppkg package
```

Comment:
Same issues as mentioned above (these should be fenced code blocks, not inline code, & shouldn't use HTML tags)